src/mpyl/utilities/s3/__init__.py

```diff
@@ -114,8 +134,8 @@ def upload_file(self, src_path: str, dst_path: str):
         :param src_path: the local relative path of the file
         :param dst_path: the destination path within the bucket
         """
-        self._logger.debug(f"Uploading to bucket: {self._buck_name}")
-        self._logger.debug(f"Uploading file at '{src_path}' to '{dst_path}'")
+        print(f"Uploading to bucket: {self._buck_name}")
+        print(f"Uploading file at '{src_path}' to '{dst_path}'")
```

Comment:
I think you forgot to revert these changes

Reply:
good catch! fixed